src/reporter/index.ts

```diff
@@ -187,15 +188,25 @@ class Reporter {
 
   // Sends a report
   async _sendReport(data) {
+    if (logFlags.debug) this.mainLogger.debug(JSON.stringify({method: '_sendReport', script: 'reporter/index', hasRecipient: this.hasRecipient}))
     if (!this.hasRecipient) {
       return
     }
     const nodeId = Self.id
+    if (logFlags.debug) this.mainLogger.debug(JSON.stringify({method: '_sendReport', script: 'reporter/index', nodeId: nodeId}))
     if (!nodeId) throw new Error('No node ID available to the Reporter module.')
     const report = {
       nodeId,
       data,
     }
+    if (logFlags.debug){
+      try {
```

Comment:
any particular reason this log is in try catch block? 
if its needed, can we add some prefix text in console.error line in catch block?

Reply:
Yes, can we add some text prefix to [this](https://github.com/shardeum/shardus-core/pull/320/files#diff-3c0b343c8b320621fa514bd5ee3e66ae28f6f5c68facd9e933f819628672f139R206) error message as identifier
we should be good to go after that

Reply:
I pushed a change to update the console.error line to have more info.